tests/cloner_test.go

```diff
@@ -51,11 +51,12 @@ var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:compo
 		}
 	})
 
-	It("[test_id:1354]Should clone data within same namespace", func() {
+	FIt("[test_id:1354]Should clone data within same namespace", func() {
```

Comment:
I think you forgot to remove the focus here.

Reply:
> I think you forgot to remove the focus here.

right, fixed